src/runtime/virtcontainers/qemu.go

```diff
@@ -2098,8 +2098,13 @@ func genericAppendBridges(devices []govmmQemu.Device, bridges []types.Bridge, ma
 				ID:   b.ID,
 				// Each bridge is required to be assigned a unique chassis id > 0
 				Chassis: idx + 1,
-				SHPC:    true,
+				SHPC:    false,
 				Addr:    strconv.FormatInt(int64(bridges[idx].Addr), 10),
+				// Reserve IO & memory windows so that
+				// we can use ACPI hotplug
+				IOReserve:     "4k",
+				MemReserve:    "1m",
```

Comment:
why 4k and not 16k ? why 1m and not 4m?  
with these sizes is possible to hotplug any devices? (vfio) 
how many devices per bridge will be supported?
sorry, this is not clear to me, could you elaborate on this?

Reply:
Ok, I've greatly expanded the comment here in the new push, I hope it answers this well enough.

Reply:
thanks :+1: